yb-voyager/cmd/import.go

```diff
@@ -203,8 +202,12 @@ func registerImportDataCommonFlags(cmd *cobra.Command) {
 	cmd.Flags().StringVar(&tableListFilePath, "table-list-file-path", "",
 		"path of the file containing the list of table names to import data")
 
+	defaultBatchSizeMsg := fmt.Sprintf("(default: target(%d), source-replica/source(%d))", DEFAULT_BATCH_SIZE_YUGABYTEDB, DEFAULT_BATCH_SIZE_ORACLE)
```

Comment:
I guess this is fine for now since we have only oracle as a potential source-replica/source. But later, we should revisit this. Ideally, we should have a different default msg based on if it's import-data-to-target or import-data-to-(source/source-replica).

Reply:
Yes.